lua/core/lazy.lua

```diff
@@ -0,0 +1,41 @@
+local lazypath = vim.fn.stdpath "data" .. "/lazy/lazy.nvim"
+if not vim.loop.fs_stat(lazypath) then
+  -- stylua: ignore
+  vim.fn.system { "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", "--branch=stable", lazypath }
+  local oldcmdheight = vim.opt.cmdheight:get()
+  vim.opt.cmdheight = 1
+  vim.notify "Please wait while plugins are installed..."
+  vim.api.nvim_create_autocmd("User", {
+    once = true,
+    pattern = "LazyInstall",
+    callback = function()
+      vim.cmd.bw()
+      vim.opt.cmdheight = oldcmdheight
+      vim.tbl_map(function(module) pcall(require, module) end, { "nvim-treesitter", "mason" })
+      astronvim.notify "Mason is installing packages if configured, check status with :Mason"
+    end,
+  })
+end
+vim.opt.rtp:prepend(lazypath)
+
+local spec = astronvim.updater.options.pin_plugins and { { import = astronvim.updater.snapshot.module } } or {}
+
+local user_plugins = astronvim.user_opts "plugins"
+for _, config_dir in ipairs(astronvim.supported_configs) do
+  if vim.fn.isdirectory(config_dir .. "/lua/user/plugins") == 1 then user_plugins = { import = "user.plugins" } end
+end
+
+vim.list_extend(spec, { { import = "plugins" }, user_plugins })
```

Comment:
I tried playing with this, but was unable to do so successfully. From what I understand `user.plugins` would be imported into `lazy.nvim`. I was wanted to know if there was a way to specify multiple places to import? like add on `user.custom.plugins` to lazy.nvim import

Reply:
You can do something like this in your `user/lazy.lua` (or in the correct place in your `user/init.lua`):

```lua
return function(config)
  vim.list_extend(config.spec, { { import = "user.custom.plugins" } })
  return config
end
```

Reply:
Thanks!